benchmark/_http-benchmarkers.js

```diff
@@ -65,7 +65,7 @@ class WrkBenchmarker {
     const args = [
       '-d', duration,
       '-c', options.connections,
-      '-t', Math.min(options.connections, require('os').cpus().length || 8),
+      '-t', Math.min(options.connections, require('os').availableParallelism() || 8),
```

Comment:
This should be reverted since, os.available parallelism is not supported in previous node versions and will remove our ability to run cross node version benchmarks.